src/join_check/mod.rs

```diff
@@ -179,7 +179,7 @@ pub async fn callback_handler(
         bot.restrict_chat_member(
             msg.chat.id,
             dlg_data.user_id,
-            ChatPermissions::SEND_MESSAGES,
+            ChatPermissions::all() - ChatPermissions::PIN_MESSAGES - ChatPermissions::MANAGE_TOPICS - ChatPermissions::CHANGE_INFO,
```

Comment:
oh, OK.
anyway we need to remove some permissions.
I didn't know how we could put them in config.toml, but it would be better to have the removals in the config file.